strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/services/impl/CronTaskExecutor.java

```diff
@@ -83,15 +83,15 @@ protected void beforeExecute(Thread t,
         }
         catch (Exception e)
         {
-            LOGGER.error(String.format("Before execute failed for [%s]", r), e);
+            LOGGER.error("Before execute failed for [{}]", r, e);
```

Comment:
Cosmetic: Please, change this to lowercase:
```suggestion
            logger.error("Before execute failed for [{}]", r, e);
```